src/settings/Settings.tsx

```diff
@@ -453,6 +453,18 @@ export const SETTINGS: { [setting: string]: ISetting } = {
         default: FontWatcher.DEFAULT_SIZE,
         controller: new FontSizeController(),
     },
+    /**
+     * With the transition to Compound we are moving to a base font size
+     * of 16px. We're taking the opportunity to move away from the `baseFontSize`
+     * setting that had a 5px offset.
+     *
+     */
+    "baseFontSizeV2": {
+        displayName: _td("Font size"),
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
```

Comment:
This should just be a device-level setting, the old one is account-level too for legacy reasons for users to not lose their old account-wide setting. The new one is only ever written to at device level as far as I can see.